src/components/detailsSubpanel.jsx

```diff
@@ -3,13 +3,24 @@
 import React from 'react';
 
 class NodeDetailsSubpanel extends React.Component {
+  constructor (props) {
+    super(props);
+    this.state = {
+      expanded: this.props.expanded ? this.props.expanded : false
+    };
```

Comment:
Good point.  I went ahead an implemented your suggestion.  I will say I had a hard time testing this because I couldn't think of any reason a parent component would want to change the state without the user clicking the thing they wanted collapse....

Oh wait, just thought of one... a collapse all, expand all type thing.

By the way, I changed the class name to match the filename and import names.  I'm using the `detailSubpanel` to display stuff in my `detailPanelConnections` and noticed that it is only called `NodeDetailsSubpanel` in its definition and nowhere else.  It just made it easier (for me) to follow who was nested under who when using the React Developer Tools browser plugins.

I've committed the changes, please review at your convenience.